README.md

```diff
@@ -1110,6 +1110,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [TenTraicion](https://github.com/TenTraicion)
   - [Thanga Mariappan](https://github.com/thangamariappan19)
   - [Tharaka Bandara](https://github.com/Ttecs)
+  - [Thirumalesh Chinna](https://github.com/ThisaruD)
```

Comment:
lgtm

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome